api/v1/testkube.yaml

```diff
@@ -564,7 +627,21 @@ components:
         executionResult:
           required: true
           description: result get from executor
-          $ref: '#/components/schemas/ExecutionResult'
+          $ref: "#/components/schemas/ExecutionResult"
+
+    Artifact:
+      type: object
+      description: API server artifact
+      properties:
+        name:
+          type: string # https://api-server/exec_id/subdirectory/file_name
+          description: artifact file path
+        size:
+          type: integer
+          description: file size in bytes
+        data:
```

Comment:
an alternative could be to not return the data inline but have a binary/octet-stream response in the operation - the client accept header would specify which to return!?